BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/file_collection/views/_file_collection_top_table.html.erb

```diff
@@ -37,8 +37,7 @@
       <%= select_all_checkbox "collection_checkbox" %>
```

Comment:
There's a bug here, it was there before this PR : the TH is not inserted in the table if the file is not synchronized, but all rows must have the same number of columns.

Reply:
Fix this unrelated bug please.